Resources/doc/cookbook/recipe_sortable_listing.rst

```diff
@@ -58,14 +58,74 @@ Then we need to inject the Sortable listener. If you only have the Gedmo bundle
                 - [ setAnnotationReader, [ "@annotation_reader" ] ]
 
 
+If you have the `Sortable behavior in admin listing
```

Comment:
Rebased, now it looks better.
I've replaced few another tabs on separate commit